src/Sylius/Bundle/CoreBundle/Command/InstallSampleDataCommand.php

```diff
@@ -68,8 +72,13 @@ protected function execute(InputInterface $input, OutputInterface $output): ?int
             return 1;
         }
 
+        $parameters = ['--no-interaction' => true];
+        if ($suite) {
+            $parameters['suite'] = $suite;
```

Comment:
Can you make explict comparition here? What do we accept as a proper value? Not empty string?